src/chapters/modules/studio.rst

```diff
@@ -0,0 +1,107 @@
+Studio: Combating Imposter Syndrome
+====================================
+
+There is a widely recognized condition called **Impostor Syndrome**. First
+described in the 1970s, it refers to a situation where someone doubts their
+accomplishments, and they fear that their success is the result of "faking it".
+People experiencing imposter syndrome ignore external evidence of their skills,
+and they attribute their success to luck.
+
+At this point in their learning journey, many LC101 students feel doubt about
+their prospects. However, they have PLENTY of company---supreme court justice
+Sonia Sotomayor, Serena Williams, Tom Hanks, and multiple CEOs have all
+questioned their success.
+
+The struggle is real, and an open conversation often helps.
+
+You CAN
+--------
+
+First, a little perspective. Identify which of the following tasks you have
+already done or know that you can accomplish:
+
+#. Use code to print "Hello, World" to the screen.
+#. Define, initialize, change, and use variables.
+#. Convert the string ``'1234'`` into a number.
+#. Construct a ``for`` loop to repeat a task 100 times.
+#. Construct ``if/else if/else`` statements to decide which of three tasks to
+   perform.
+#. Build, modify, and access an array.
+#. Design and call a function.
+#. Call one function from within another function.
+#. Find and fix bugs in a segment of non-working code.
+
+How many of the 9 items listed above did you indicate? There is no 'passing'
+score for this. Whether you checked all 9 or only 1 or 2, simply saying, *I can
+do that*, means you have more coding skill than the bulk of the world's
+population.
+
+Doubt and uncertainty are normal, especially when exploring a new career.
+However, with the skills you already know, you can legitimately say, *I am a
+coder*. Combined with the skills you will learn during the rest of the course,
+there can be no doubt. You ARE NOT pretending.
+
+Discussion
+-----------
+
+Take a few moments in the studio to consider, share, and discuss the following:
+
+#. Have you ever felt the effects of Imposter Syndrome? When?
+#. Have you ever responded to a compliment by diminishing the work that earned
+   you the praise? If so, why did you answer in that way?
+#. How do you feel in a test/quiz/studio when someone finishes much earlier
+   than you?
+#. What are you most proud of from your time working in LC101?
```

Comment:
Remove mention of LC101 so content can be used in other programs.

Reply:
Done.